git/commits_test.go

```diff
@@ -137,8 +150,13 @@ func TestGetConventionalCommitTypesSinceLatestRelease(t *testing.T) {
 		}
 
 		assert.NoError(t, err)
-		assert.Equal(t, test.expectedLastVersion, actual.LatestReleaseVersion)
+
+		// We need to compare using semver's own Equal function, because under
```

Comment:
I don't think this comment is necessary. It might not be intuitive why you need it, but when reading the code it is immediately clear what happens and that is what matters.

Reply:
I am also fine with keeping it, since it is just in a test.

Reply:
Since the reasoning for this comment is tightly coupled to my reasoning for having both assertions, I'll close this one. If we get rid of one of the two assertions, the comment can be removed as well.